tests/integration_tests.rs

```diff
@@ -41,70 +41,178 @@ struct Position {
-    ]);
+            baz",
+        vec![
+            Position {
+                line: 1,
+                column: 2,
+                offset: 1,
+                fragment_len: 3,
+            },
+            Position {
+                line: 2,
+                column: 9,
+                offset: 13,
+                fragment_len: 3,
+            },
+            Position {
+                line: 3,
+                column: 13,
+                offset: 29,
+                fragment_len: 3,
+            },
+            Position {
+                line: 3,
+                column: 16,
+                offset: 32,
+                fragment_len: 3,
+            },
+        ],
+    );
 }
 
 #[test]
 fn it_locates_u8_fragments() {
-    test_str_fragments(simple_parser_u8, &b"foobarbaz"[..], vec![
-        Position { line: 1, column: 1, offset: 0, fragment_len: 3 },
-        Position { line: 1, column: 4, offset: 3, fragment_len: 3 },
-        Position { line: 1, column: 7, offset: 6, fragment_len: 3 },
-        Position { line: 1, column: 10, offset: 9, fragment_len: 3 }
-    ]);
-    test_str_fragments(simple_parser_u8, &b" foo
+    test_str_fragments(
+        simple_parser_u8,
+        &b"foobarbaz"[..],
+        vec![
+            Position {
+                line: 1,
+                column: 1,
+                offset: 0,
+                fragment_len: 3,
+            },
+            Position {
+                line: 1,
+                column: 4,
+                offset: 3,
+                fragment_len: 3,
+            },
+            Position {
+                line: 1,
+                column: 7,
+                offset: 6,
+                fragment_len: 3,
+            },
+            Position {
+                line: 1,
+                column: 10,
+                offset: 9,
+                fragment_len: 3,
+            },
+        ],
+    );
+    test_str_fragments(
+        simple_parser_u8,
+        &b" foo
         bar
-            baz"[..], vec![
-        Position { line: 1, column: 2,  offset: 1,  fragment_len: 3 },
-        Position { line: 2, column: 9,  offset: 13, fragment_len: 3 },
-        Position { line: 3, column: 13, offset: 29, fragment_len: 3 },
-        Position { line: 3, column: 16, offset: 32, fragment_len: 3 }
-    ]);
+            baz"[..],
+        vec![
+            Position {
+                line: 1,
+                column: 2,
+                offset: 1,
+                fragment_len: 3,
+            },
+            Position {
+                line: 2,
+                column: 9,
+                offset: 13,
+                fragment_len: 3,
+            },
+            Position {
+                line: 3,
+                column: 13,
+                offset: 29,
+                fragment_len: 3,
+            },
+            Position {
+                line: 3,
+                column: 16,
+                offset: 32,
+                fragment_len: 3,
+            },
+        ],
+    );
```

Comment:
I prefer the old formatting. The reason why is that we can read the difference more easily: each `Position` properties are aligned in columns, which emphasizes the difference of the values.

Do you mind putting it back?

(the other formatting changes are OK for me otherwise :))